MyCryptoHQ/MyCrypto
# Enable back button from "active wallet" to "select a wallet"

Closes #1768

### Description
Because the homepage immediately redirects to `/account`, there is never an initial entry made to go back to.

### Changes

* The initial redirect from `/` to `/account` now pushes an entry onto the history.

### Steps to Test

1. Visit the homepage.
2. Access a wallet.
3. Press the back button and expect a log-out prompt to occur.
4. Be back at the "select a wallet" homepage.

### Screenshots
<img width="1438" alt="screen shot 2018-06-21 at 4 42 40 pm" src="https://user-images.githubusercontent.com/10479826/41747022-258fb6c2-7572-11e8-909c-cacdbdbb64d3.png">
